lang/main.json

```diff
@@ -428,5 +428,13 @@
         "noOtherDevices": "No other devices available",
         "selectADevice": "Select a device",
         "testAudio": "Test sound"
+    },
+    "invite": {
+        "addPassword": "Add password",
+        "hidePassword": "Hide password",
```

Comment:
Can we make sure that strings that are not used anymore are deleted. I see:
"addPassword": "Add a password",
"createPassword": "Create password",
in the same file.